.env.tpl

```diff
@@ -2,3 +2,5 @@ OAUTH_CLIENT_ID=
 OAUTH_CLIENT_SECRET=
 JWT_SIGNING_KEY=testing
 DB_CONNECTION=sqlite:///path/to/db.db
+OAUTH_RESTRICT_ACCESS=
+OAUTH_RESTRICT_REVIEWER_ACCESS=
```

Comment:
`Reviewer` instead of `Requesters`?
I don't see the reason because this PR introduces a new concept colliding with an already existing one. We decide to call that role `requester`, so it would be a good idea to stick to our convention,
I'd suggest doing `s/reviewer/requester/g` before merging.

Reply:
👍 for consistency.

Let's rename and merge, with subsequent improvements planned in #72

Reply:
let's try to use correct names everywhere. In the issue description, it's called `reviewers`.

Reply:
It might be a misunderstanding when the latest issue was created.
In the [schema definition issue](https://github.com/src-d/code-annotation/issues/2) and in our code, we're already using `requester`.